languages/java/exercises/concept/classes/.docs/introduction.md

```diff
@@ -0,0 +1,66 @@
+The primary object-oriented construct in Java is the _class_, which is a combination of data (_fields_) and behavior (_methods_). The fields and methods of a class are known as its _members_.
+
+Access to members can be controlled through access modifiers, the two most common ones being:
+
+- `public`: the member can be accessed by any code (no restrictions).
+- `private`: the member can only be accessed by code in the same class.
+
+You can think of a class as a template for creating instances of that class. To create an instance of a class (also known as an _object_), the `new` keyword is used:
+
+```csharp
+class Car
+{
```

Comment:
normally in Java left curly bracket is on the same line of the class name
so 
```
class Car {
}
```
This is valid for all the code in this page
This is valid for block of code too